pkg/didcomm/common/service/event.go

```diff
@@ -102,7 +102,7 @@ type Event interface {
 //	go service.AutoExecuteActionEvent(actionCh)
 func AutoExecuteActionEvent(ch chan DIDCommAction) error {
 	for msg := range ch {
-		msg.Continue()
+		msg.Continue(nil)
```

Comment:
Not a fan of nil arguments, particularly when passing nil into an interface.

Reply:
I think we should either:
- Create a specific Empty type (and use instead of nil).
- Use two methods.

Reply:
Done